app/db/models/user.js

```diff
@@ -228,13 +243,16 @@ module.exports = (Sequelize, db) => {
       firstName: this.getDataValue('firstName'),
       lastName: this.getDataValue('lastName'),
       profilePicture: this.getDataValue('profilePicture'),
+      graduationYear: this.getDataValue('graduationYear'),
+      major: this.getDataValue('major'),
       points: this.getDataValue('points'),
     };
   };
 
   User.prototype.getUserProfile = function () {
     return {
       uuid: this.getDataValue('uuid'),
+      accountType: this.getDataValue('accessType'),
```

Comment:
same here with merge conflicts

Reply:
I added? Or you added? I don't see any merge conflicts with master, but this is needed so front end knows what access a user has

Reply:
looks like this is from `master` (see [here](https://github.com/acmucsd/membership-portal/blob/master/app/db/models/user.js#L240))